src/configuration/configuration.json

```diff
@@ -0,0 +1,12 @@
+{
+    "plugin_preferences":{
```

Comment:
add default too.
I think default is a mandatory property in this object.

Reply:
done